subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,98 @@
+
+#### Comprehension
+
+##### Ask the student to describe the microservices architecture implemented.
+
+###### Have you clearly defined the boundaries for each microservice based on business domains?
+
+###### Do your microservices align directly with specific business functions?
+
+###### Are your microservices designed to operate independently of one another?
+
+###### Can each microservice be deployed, updated, and scaled without affecting others?
+
+###### Is your architecture designed to support the independent scalability of each microservice?
+
+###### Does your system maintain functionality even when one or more services fail?
+
+###### Is there an API Gateway in your architecture to manage incoming requests?
```

Comment:
This is not explicitly required in the subject - are you considering this to be a "known" best practice to design a microservices application? Should this be added as a requirement in the subject?

For example, in section 1, `Configure a Microservices Architecture`
```
...
* Ensure microservices can only be accessed through an API Gateway 
```
Or something similar

Reply:
a microservices system without a gateway is not a microservices system. so when i ask them to make microservices this automatically include a gateway to be present.